test/controllers/user.test.js

```diff
@@ -0,0 +1,68 @@
+const supertest = require('supertest');
+const models = require('../../app/models/index');
+const app = require('../../app');
+
+const request = supertest(app);
+const userAttributes = {
+  firstName: 'Omar',
+  lastName: 'Rodriguez',
+  email: 'omar.rodriguez@wolox.com',
+  password: 'password1923'
+};
+
+describe('usersController.signUp', () => {
+  it('Creates a user', () =>
+    request
+      .post('/users')
+      .send(userAttributes)
+      .then(response => {
+        expect(response.text).toBe('Omar');
+      })
+      .then(() =>
+        models.user
+          .findOne({
+            where: {
+              firstName: 'Omar',
+              lastName: 'Rodriguez',
+              email: 'omar.rodriguez@wolox.com'
+            }
+          })
+          .then(user => {
+            expect(user.email).toBe('omar.rodriguez@wolox.com');
+          })
+      ));
+
+  it('Try to create a existing user and fails', () =>
+    request
+      .post('/users')
+      .send(userAttributes)
+      .then(() =>
+        request
+          .post('/users')
+          .send(userAttributes)
+          .then(response => expect(JSON.parse(response.text).internal_code).toBe('email_already_in_use'))
```

Comment:
You should not have to parse `response.text`. Check if you can access the property `internal_code` directly.  Should be in either `response.body` or `response.response.body`